editor/gui/editor_run_bar.cpp

```diff
@@ -157,6 +167,23 @@ void EditorRunBar::_update_play_buttons() {
 	}
 }
 
+void EditorRunBar::_movie_maker_item_pressed(int p_id) {
+	switch (p_id) {
+		case TOGGLE_MOVIE_MAKER: { // Toggle Movie Maker mode.
```

Comment:
```suggestion
		case TOGGLE_MOVIE_MAKER: {
```
No reason to repeat the enum.

Reply:
Makes sense, will do!